packages/hooks/tests/useOrganizations.test.js

```diff
@@ -31,10 +29,11 @@ describe('useOrganizations', () => {
       'An error occurred'
     );
 
-    const { getByText } = render(<Component />);
+    const { getByTestId } = render(<Component />);
 
-    await waitForElement(() => getByText('"An error occurred"'));
+    await waitForElement(() => getByTestId('invalid'));
   });
+
```

Comment:
The test below on line 37 should be `async` and the `getByTestId('loading')` should be `await`ed, I think that's why the test results are varying based on when and how many tests are run. This should also fix the tests for `useCurrentRegion` and `useCurrentUser` since they have a similar non-async test